src/notation/inotationinteraction.h

```diff
@@ -84,6 +84,12 @@ class INotationInteraction
     virtual async::Notification textEditingStarted() const = 0;
     virtual async::Notification textEditingChanged() const = 0;
 
+    // Grip edit
+    virtual bool isGripEditStarted() const = 0;
+    virtual bool isClickOnGrip(const QPointF& clickPos) = 0;
```

Comment:
I think, better to rename those arguments to logicPos or just pos (as above)

Reply:
isClickOnGrip should be const

Reply:
fixed